images/inference-services/kserve/templates/ifs.lib.yaml

```diff
@@ -0,0 +1,17 @@
+#@ def name(inferenceService):
+#@  if inferenceService.appName:
+#@    return inferenceService.appName
+#@  else:
+#@    return inferenceService.project + "-" + inferenceService.codeset + "-" + inferenceService.workflow
+#@  end
+#@ end
```

Comment:
Since you're always setting appName to $APP_NAME, and APP_NAME is never empty, this extra logic is not useful anymore. You should remove it.

Reply:
You are right. I think I forgot to revisit it after you introduced `app_name`. I will remove it.